inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/impl/tube/InLongTubeFetcherImpl.java

```diff
@@ -174,6 +174,15 @@ public long getAckedOffset() {
         return 0L;
     }
 
+    /**
+     * isValidState
+     */
+    public void isValidState() {
```

Comment:
why not return false, but throw exception in the method called isxxx when the state is not valid.

Reply:
This method has not been completed yet